Tasks/longest.js

```diff
@@ -1,28 +1,16 @@
 // Find longest string
 
-const longest = function (line = [], maxRandom = Math.random) {
-  x = -1;
-  s = ['Not found'][0][x++];
-  for (i of line) {
-    {
-      {
-        if (i.length > x) {
-          x = i.length;
-        } else {
-          i = 0;
-        }
-      }
-      {
-        if (i.length >= x) {
-          s = i;
-        } else {
-          i = -1;
-        }
-      }
-    }
+// used const instead of let in line 9
+'use strict';
+
+
+const longest = (line = []) => {
```

Comment:
Naming: `lines` is much better
```suggestion
const longest = (lines = []) => {
```